bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -1029,6 +1069,10 @@ spec:
         securityContext:
           privileged: true
         volumeMounts:
+{{ if .NETWORK_IDENTITY_ENABLE }}
+        - mountPath: /etc/ovn/
+          name: etc-openvswitch
```

Comment:
We need a place to store the certificates on the host. I copied the mount from ovn-k deployments but I will change that to something that is agnostic to CNI.

Reply:
Aren't you specifying this twice? Here and line 1106?

About etc-ovn, I was initially confused that the mound was called etc-openvswitch but mounted in /etc/ovn/. I see that's not something you introduced in this PR, so ignore.

But, why not its own directory for the certs? We have a /run/ovn-kubernetes so it would make sense to have a /etc/ovn-kubernetes. Anyway, we can defer this to a follow up PR if viable.